File: mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    //Ask for height (1 to 8 include)
   int n;
    do
    {
        n = get_int("height: ");
    }
    while (n > 8 | n < 1);

    //print #

    int j = 0;
    while (j < n)
    {
        printf("#");
        printf("\n");
        j++;
    }

}

File: cs50x/week6/pset6/sentimental-mario-less/mario.py
# TODO

from cs50 import get_int


# get height
def main():
    height = get_height()
    printp(height)


# print space
def printp(n):
    row = n
    for i in range(n):
        print(" " * (row - 1), end="")
        print("#" * (i + 1))
        row -= 1


# get height from user
def get_height():
    while True:
        n = get_int("height: ")
        if n > 0 and n < 9:
            return n


main()
